src/components/Package.tsx

```diff
@@ -13,43 +17,102 @@ interface PackageState {
     packageName?: string
```

Comment:
Probably worth starting to actually define the types, rather than just using any